sockeye/rnn.py

```diff
@@ -12,7 +12,7 @@
 # permissions and limitations under the License.
 
 # List is needed for mypy, but not used in the code, only in special comments
-from typing import Optional, List
+from typing import Optional, List, Iterable # NOQA pylint: disable=unused-import
```

Comment:
Because List is only used in a comment (intended for mypy, if I'm not mistaken), and thus pylint and flake8 (->NOQA) complain about an unused import.